topbeat/system/sigar_test.go

```diff
@@ -1,6 +1,6 @@
 // +build !integration
```

Comment:
Wouldn't make sense to split the unit tests to multiple files as well, like system_test, process_test & filesystem_test?